dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasableResourceInterface.java

```diff
@@ -49,27 +54,36 @@
      */
     T getAndCheckResourceByAlias(String alias);
 
+
+    static void checkAliases(Set<String>  aliases, User user) {
```

Comment:
Requires Javadoc, but the bigger issue is how does this stop someone from pre-claiming zenodo DOIs by calling the endpoint to create aliases that look like zenodo DOIs ahead of time and guessing them ahead of time?

Reply:
@denis-yuen Is this a situation where some bad actor would create many alias permutations of the form 10.####-zenodo.####### so that Dockstore would not ever be able to register a DOI for that workflow? If so then maybe we should prevent creating aliases of that form from the Dockstore endpoint except when Dockstore tries to issue a DOI?  Thanks for clarifying!

Reply:
Yup, exactly